### PR TITLE
Add LeetCode

### PR DESCRIPTION
**Pull request by virusvn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +58,8 @@ Feel free to send a PR! :smile:
 - Fix a [bug](https://github.com/npm/npm/issues), get a fashionable pair of socks! [Reference](http://blog.npmjs.org/post/129827785565/npm-weekly-30-package-scripts-for-tooling-a)
 
 ![](https://partners.npmjs.com/weekly/weekly30/socks-1100x.jpg)
+
+# LeetCode
+
+- Solve problems to earn LeetCode Point and get t-shirt
+![](https://leetcode.com/static/images/store/leetcode_kit_promo.png)
```
Comment (appleboy) [inline README.md]: Add link?

**Comment by samuelbeard** [comment]
Can I suggest a bit of grammar fixing on line 64?

`Solve problems to earn LeetCode Point and get t-shirt [LeetCode](http://leetcode.com/)`

could be fixed to:

`Solve problems to earn LeetCode Points and get a t-shirt [LeetCode](http://leetcode.com/)`

**Comment by championshuttler** [comment]
@virusvn how many points we need to earn and how to use those points?

**Comment by virusvn** [comment]
We need 6000 points to get a t-shirt, it's hard!
Here is some cases we can earn points:

| Case | Points | Time |
| ------ | --- | ---- |
| Contribute a Question | 200 | Anytime |
| Contribute a Testcase | 200 | Anytime |
| 1st Place in a Contest  | 5000 | Every week |
| 2nd Place in a Contest | 2500 | Every week |
| 3rd Place in a Contest | 1000 | Every week |
| Top 50 in a Contest | 300       | Every week |
| Top 100 in a Contest | 100       | Every week |
| Top 200 in a Contest | 50       | Every week |

**Comment by swapagarwal** [comment]
I'm cleaning up all old issues and PRs which did not get much traction. The popular ones have already been merged. I'm focusing on getting a website ready before Hacktoberfest. 😇

**Pull request closed without merging**